### ISSUE TITLE
Seems like getCapital() functionality not works

### ISSUE THREAD

**Issue by mikolazp** [MenaraSolutions/geographer#16]
this code don't return any value
$capital = $thailand->getCapital();

I'm check geographer-data dir, and seems like there not exist any info about capital of countries.

**Comment by dusterio** [comment]
yes, you are right :) I planned it but never added it. It's probably the only feature mentioned in the README that doesn't exist.

It's pretty easy to implement though

**Comment by mikolazp** [comment]
> It's pretty easy to implement though
Probably yes, but unfortunately I didn't get, where I can get this info in json files.
This file not contain such info
vendor/menarasolutions/geographer-data/resources/countries.json
